lib/Request.js

```diff
@@ -110,8 +118,9 @@ Request.prototype.getParam = function(param) {
     return this._params.hasOwnProperty(param) ? this._params[param] : undefined;
 };
 
-Request.prototype.setQueries = function(queries) {
-    this._url.query = queries;
+Request.prototype.setQueries = function(query) {
```

Comment:
What type should `query` be? If it should be an object, let's enforce it. If the object needs to fit a particular structure, let's wrap it in a class.